config.go

```diff
@@ -33,6 +33,11 @@ type configuration struct {
 		Kafka ingestKafkaConfiguration
 	}
 
+	// Dedupe configuration
+	Dedupe struct {
```

Comment:
We could provide a simple in-memory implementation for local development and testing. Then the enable flag should be moved the dedupe level.

I'm also wondering if this should be within ingest, given that it's happening during ingest.

Reply:
Added